sites/public/pages/applications/financial/vouchers.tsx

```diff
@@ -125,14 +125,20 @@ const ApplicationVouchers = () => {
             <fieldset>
               <legend className="sr-only">{t("application.financial.vouchers.legend")}</legend>
               <FieldGroup
+                fieldGroupClassName="grid grid-cols-1"
+                fieldClassName="ml-0"
                 type="radio"
                 name="incomeVouchers"
                 error={errors.incomeVouchers}
                 errorMessage={t("errors.selectAnOption")}
                 register={register}
-                validation={{ required: true }}
                 fields={incomeVouchersValues}
                 dataTestId={"app-income-vouchers"}
+                validation={{
```

Comment:
Anywhere this change happens - we were not resetting the error state once the user selected a value